02_flow_compensation/exercises.m

```diff
@@ -11,35 +11,33 @@
 dosedata = '../example_assay/';
 
 
-
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-% Examples of spectral overlap:
+% Examples of spectral overlap (Fig1 to Fig15):
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 % Let's look at some single-color positive controls:
 % no significant spectral overlap
-fcs_scatter([colordata '07-29-11_EYFP_P3.fcs'],'FITC-A','Pacific Blue-A',1,[0 0; 6 6],1);
+fcs_scatter([colordata '07-29-11_EYFP_P3.fcs'],'FITC-A','Pacific Blue-A',1,[0 0; 6 6],1); % Fig1
 % minor spectral overlap
-fcs_scatter([colordata '07-29-11_EYFP_P3.fcs'],'FITC-A','PE-TxRed YG-A',1,[0 0; 6 6],1);
+fcs_scatter([colordata '07-29-11_EYFP_P3.fcs'],'FITC-A','PE-TxRed YG-A',1,[0 0; 6 6],1); % Fig2
 % significant spectral overlap
-fcs_scatter([colordata '07-29-11_mkate_P3.fcs'],'PE-TxRed YG-A','FITC-A',1,[0 0; 6 6],1);
-fcs_scatter([colordata '07-29-11_EYFP_P3.fcs'],'FITC-A','AmCyan-A',1,[0 0; 6 6],1);
+fcs_scatter([colordata '07-29-11_mkate_P3.fcs'],'PE-TxRed YG-A','FITC-A',1,[0 0; 6 6],1); % Fig3
+fcs_scatter([colordata '07-29-11_EYFP_P3.fcs'],'FITC-A','AmCyan-A',1,[0 0; 6 6],1); % Fig4
 % massive spectral overlap
-fcs_scatter([colordata '07-29-11_EBFP2_P3.fcs'],'Pacific Blue-A','AmCyan-A',1,[0 0; 6 6],1);
+fcs_scatter([colordata '07-29-11_EBFP2_P3.fcs'],'Pacific Blue-A','AmCyan-A',1,[0 0; 6 6],1); % Fig5
 
-% let's look at some of these without blending:
-fcs_scatter([colordata '07-29-11_EYFP_P3.fcs'],'FITC-A','PE-TxRed YG-A',0,[0 0; 6 6],1);
-fcs_scatter([colordata '07-29-11_EYFP_P3.fcs'],'FITC-A','AmCyan-A',0,[0 0; 6 6],1);
-fcs_scatter([colordata '07-29-11_EBFP2_P3.fcs'],'Pacific Blue-A','AmCyan-A',0,[0 0; 6 6],1);
+% let's look at some of these without blending (density set to 0):
```

Comment:
Let's clarify a little more --- "plot 'density' set to zero"

Reply:
changed